Assets/Plugins/Source/Core/EOSManager.cs

```diff
@@ -1378,9 +1378,10 @@ public void StartLoginWithLoginOptions(LoginOptions loginOptions, OnAuthLoginCal
-                EOSAuthInterface.Login(ref modifiedLoginOptions, null, (Epic.OnlineServices.Auth.OnLoginCallback)((ref Epic.OnlineServices.Auth.LoginCallbackInfo data) => {
+                IOSLoginOptions modifiedLoginOptions = EOS_iOSLoginOptionsHelper.MakeIOSLoginOptionsFromDefualt(loginOptions);
```

Comment:
Looks like you still have the spelling mistake here, should be `MakeIOSLoginOptionsFromDefault`

Reply:
Right, I forgot to fix this and the actual function. Thanks!

Reply:
Fixed!